README.md

```diff
@@ -4,45 +4,194 @@
+
+Go ahead and build and run this app. Tap on an item in the main view. You'll see another view slide in from the right with some details about the fruit. You'll also see the title (in the nav bar) change to reflect the name of that fruit, and you'll also see a back button so you can go back to the main listing.
+
+How are these views tied together?
+
+Open `Main.storyboard` in Interface Builder again. Notice that there is a little icon connecting the two views together:
+
+![Segue icon](https://s3.amazonaws.com/learn-verified/segue-icon.png)
+
+This icon represents a _segue_. You can click on it and view its attributes in Interface Builder's Attributes Inspector. In particular, note that its _identifier_ has been set to "ShowFruitDetails". This string will be used to identify the segue in your Swift code. Later on, we'll cover in more detail how you create segues between views yourself in Interface Builder; for now, it's important to learn how you can identify segues and work with them.
+
+You know that there's a segue between the `FruitListingViewController` (represented as **Fruits Scene** in the storyboard) and `FruitViewController` (represented as **Fruit View Controller Scene** in the storyboard). Open up `FruitListingViewController.swift` in Xcode's editor. If you scroll to the bottom of the file, you'll find a method called `prepareForSegue(_:sender:)` that looks like this:
+
+```swift
+override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
+    if segue.identifier != "ShowFruitDetails" { return }
+    if let dest = segue.destinationViewController as? FruitViewController,
+           indexPath = tableView.indexPathForSelectedRow {
+        dest.fruit = fruits[indexPath.row]
+    }
+}
+```
+
+That's where the segue magic happens.
+
+This method is called whenever your iOS app is transitioning from the main view (backed by `FruitListingViewController`) to another view. In other words, every time you tap an item in the fruit listing in the main view, this method is called. It gets passed a `UIStoryboardSegue` object, which represents the segue (the orange lines in the diagram shown earlier in this lesson). That `segue` object contains some important data that faciliates the transition from one view to another.
+
+`prepareForSegue(_:sender:)` first checks to see if you're dealing with the segue from the main view to the fruits detail view. If you're not, the method does nothing and returns immediately. (You should always be dealing with that segue, since it's the only one you've set up, but it's good to get into the habit to check anyway.)
```

Comment:
I will add this myself.

I think we can say something like this on line 104 instead of starting off with `prepareForSegue(_:sender:)` to rid any _possible* confusion that this method is doing this on its own (I doubt that conduction will exist, maybe I'm over-thinking this). 

"Our implementation of `prepareForSegue(_:sender:)` here first checks*